hushline/settings/__init__.py

```diff
@@ -274,6 +274,16 @@ async def index() -> str | Response:
             )
             user_count = len(all_users)
 
+        # Load the business tier price
+        business_tier = db.session.query(Tier).filter_by(name="Business").first()
```

Comment:
```suggestion
        business_tier = db.session.scalars(db.select(Tier).filter_by(name="Business")).one_or_none()
```

Reply:
I noticed this seems to be the only place where we lookup the business tier by name instead of ID. Is that intentional?

Reply:
Nope that wasn't intentional. Fixed with https://github.com/scidsg/hushline/pull/595/commits/ebf6c998e3265785ed23bc94ca63326f198331e9